examples/flows/hello-world-yaml/definition.yaml

```diff
@@ -0,0 +1,14 @@
+Comment: Run the Hello World Action as the Flow's identity
```

Comment:
I agree with @urimandujano 's comment on the call. This does read more cleanly than the equivalent JSON to me.  Seems to validate the general idea and makes the rest of the effort worth it.

If nothing else, the LoC count goes down compared to the JSON equivalent which is not a bad thing.